### PR TITLE
feat: float filter button on mobile collection activity page

### LINKED ISSUE #5359: Mobile collection activity menu
image
- we are going to put fixed button on mobile for collection activity 
- that means it will be easier for phone users to apply filters and dont have to scroll all the way up
- button will be fixed only on activity tab, on items the layout stays the same
- next to the filter - we will show number of active filters
- if you click - menu is opened 

https://www.figma.com/file/7BWdGiowrvTucpSFLE0kRX/Collection-detail-handoff?node-id=0%3A1&t=Cuy1OtrN6bTUjwvS-1
- updated here

follow up issue for 
- #5272

### PR DESCRIPTION
**Pull request by leo-anderson-x**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #5359 
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [x] I've tested it at </bsx/collection>
- [x] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DzUbHCk3Fr3XdCPNKo7uCJvtBH7YfgiFbn4Gr3VmCMiFy1C)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

https://user-images.githubusercontent.com/128157824/230609671-444e8387-84b0-40ec-95c3-48b502dfba2f.mp4

### CODE REVIEW

File: components/explore/FilterFloatButton.vue [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+</template>
+
+<script setup lang="ts">
+import { NeoButton } from '@kodadot1/brick'
+import { usePreferencesStore } from '@/stores/preferences'
+
+const route = useRoute()
+const preferencesStore = usePreferencesStore()
+
+const numOfActiveFilters = computed(() => {
```
Comment (daiagi) [inline components/explore/FilterFloatButton.vue:17]: this is becoming a shared functionality shared by FilterMeunButton, this component and BreadCrumbsFilter
please extract it to a composable

File: components/explore/MobileControls.vue [hunk 2]
```diff
@@ -2,7 +2,8 @@
   <div class="explore is-flex is-flex-wrap-wrap">
     <ExploreTabs />
     <div class="explore-menu is-flex">
-      <FilterMenuButton />
+      <FilterMenuButton v-if="!isActivityTab" />
+      <FilterFloatButton v-if="isActivityTab" />
```
Comment (daiagi) [inline components/explore/MobileControls.vue:6]: please use v-else instead

File: components/explore/FilterFloatButton.vue [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    ([key, value]) => (key === 'search' && Boolean(value)) || value === 'true'
+  )
+
+  return activeFilters.length
+})
+const openMobileFilters = () => preferencesStore.setMobileFilterCollapse(true)
+</script>
+
+<style lang="scss" scoped>
+@import '@/styles/abstracts/variables';
```
Comment (daiagi) [inline components/explore/FilterFloatButton.vue:29]: not needed as you are not using any theme style here

**Comment by exezbcz** [comment]
good! 
can we please make the space disappear after the burger menu is not displayed? - I mean the space between tab switch and white line
image

**Review by daiagi — CHANGES_REQUESTED** [review]
few changes please

**Comment by leo-anderson-x** [comment]
> good! can we please make the space disappear after the burger menu is not displayed? - I mean the space between tab switch and white line

@exezbcz fixed:
deploy-preview-5546--koda-canary netlify app_bsx_collection_3586573150_activity_page=1(iPhone SE)

**Comment by exezbcz** [comment]
@leo-anderson-x yes, good job!

File: components/explore/MobileControls.vue [hunk 4]
```diff
@@ -18,6 +21,7 @@ import ExploreSort from './ExploreSort.vue'
 import ExploreChain from './ExploreChain.vue'
 import ExploreOffer from './ExploreOffer.vue'
 import FilterMenuButton from './FilterMenuButton.vue'
+import FilterFloatButton from '../collection/activity/FilterFloatButton.vue'
```
Comment (daiagi) [inline components/explore/MobileControls.vue:24]:
```suggestion
import FilterFloatButton from '@/components/collection/activity/FilterFloatButton.vue'
```

**Review by daiagi — COMMENTED** [review]
nice job!
just fix 1 relative import please

**Review by daiagi — APPROVED** [review]

**Comment by yangwao** [comment]
pay 30 usd

**Pull request merged**